examples/helm/101_initial_cluster.yaml

```diff
@@ -1,4 +1,5 @@
 topology:
+  provider: k8s
```

Comment:
If we're defaulting to k8s, should we leave these lines out?

Reply:
I vote to leave it in, because it serves to document the field and the default value.

Reply:
That'll be documented in the main helm values.yaml file. Putting it in all these example files just adds more mental complexity for somebody getting started. There's enough to take in with Vitess that it's nice to not force people to think about their choice of topo server right out of the gate.